File: Modul-3/Soal-4/PRAK304-2410817210008-AhmadUlyani.c
#include <stdio.h>
int main(){
    int x;
    scanf("%d", &x);
    if (x == 0){
        printf("Nol");
    }
    else if (x > 0 && x < 10){
        printf("Satuan");
    }
    else if (x > 10 && x < 20){
        printf("Belasan");
    }
    else if (x >= 10 && x < 100){
        printf("Puluhan");
    }
    else {
        printf("Anda menginput melebihi limit bilangan");
    }

    return 0;
}

File: Modul-3/Soal-4/PRAK304-2410817210008-AhmadUlyani.py
x = int(input())
if x == 0:
    print("Nol")
elif 0 < x < 10:
    print("Satuan")
elif 10 < x < 20:
    print("Belasan")
elif 10 <= x < 100:
    print("Puluhan")
else:
    print("Anda Menginput Melebihi Limit Bilangan")